code/game/gamemodes/nations/nationsprocs.dm

```diff
@@ -9,6 +9,7 @@
 	if(!mode.kickoff) return 1
 
 	var/mob/living/carbon/human/H = src
+	if(H.stat==2) return
```

Comment:
Should really check for `stat == DEAD` rather than 2, they're effectively the same but the latter is a bit more clear/consistent.